userspace/libscap/scap.c

```diff
@@ -269,7 +269,9 @@ scap_t* scap_open_live_int(char *error,
 	if((res = scap_proc_scan_proc_dir(handle, filename, -1, -1, NULL, error, true)) != SCAP_SUCCESS)
 	{
 		scap_close(handle);
-		snprintf(error, SCAP_LASTERR_SIZE, "error creating the process list. Make sure you have root credentials.");
+		char err[SCAP_LASTERR_SIZE] = {0};
```

Comment:
Yeah that's right, although usually `strncpy` comes in a pattern where you either manually terminate the string after strncpying or you allocate one byte more and place a 0 there before the strncpy. That's why I noticed.

Just a small detail to avoid problems in case things will change in the future inside `scap_proc_scan_proc_dir`.